README.md

```diff
@@ -28,6 +28,13 @@ coraza-spoa -f /etc/coraza-spoa/coraza-spoa.yaml
 
 You will also want to download & extract the [OWASP Core Ruleset]( https://github.com/coreruleset/coreruleset/releases) (version 4+ supported) to the `/etc/coraza-spoa` directory.
 
+### Validate Configuration
+To validate the configuration, you can pass the `-validate` flag, which just loads the configuration and exits.
```

Comment:
I really don't like our current flag names but because renaming would be a breaking change I left them as they are. I would really like to use haproxy like flags e.g. `-c` for check mode and supplying the config file as first argument to the process. What do you think about this?

Reply:
You have changed in the example above already to the flag `-f`
I added it to have long and short versions possible